authentication/build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
 description 'authentication: Deephaven authentication and identity'
 
 dependencies {
+    api project(':Util')
```

Comment:
fwiw, it used to be that AuthContext was in `:Util`, so it was necessary to export this. Now we need to have `:Base` instead, so that LogOutputAppendable can be exposed.